modules/home-manager/hyprland.nix

```diff
@@ -14,25 +13,14 @@ in
 
   config = lib.mkIf enable {
     home.sessionVariables = lib.mkIf pointerCursor.enable {
-      HYPRCURSOR_SIZE = "24";
+      HYPRCURSOR_SIZE = config.home.pointerCursor.size;
       HYPRCURSOR_THEME = "catppuccin-${pointerCursor.flavor}-${pointerCursor.accent}-cursors";
     };
 
     wayland.windowManager.hyprland.settings = {
-      source =
-        [
-          "${sources.hyprland}/themes/${cfg.flavor}.conf"
-          (builtins.toFile "hyprland-${cfg.accent}-accent.conf" ''
-            $accent=''$${cfg.accent}
-            $accentAlpha=''$${cfg.accent}Alpha
-          '')
-        ]
-        ++ lib.optionals pointerCursor.enable [
-          (builtins.toFile "hyprland-cursors.conf" ''
-            env = HYPRCURSOR_THEME,MyCursor
-            env = HYPRCURSOR_SIZE,24
-          '')
-        ];
+      source = "${sources.hyprland}/themes/${cfg.flavor}.conf";
+      "$accent" = "\$${cfg.accent}";
+      "$accentAlpha" = "\$${cfg.accent}Alpha";
```

Comment:
These values were provided as a file to source because nix may decide to define variables of the user before accent colors. Thus, user may not be able to use those accent colors in custom variables

Example:
```nix
wayland.windowManager.hyprland.settings = {
  # inside ctp
  "$accent" = "\$${cfg.accent}";
  # user config
  "$AAAmyvar" = "<span color="$accent">...</span>";
};
```
Could result to
```ini
source=/nix/store/...-hyprland/themes/...
$AAAmyvar = <span color="$accent">...</span> # oops, $accent is an empty string
$accent = $magenta
# ...
```

Reply:
I'll revert sometime

Reply:
#347